src/main/java/com/google/jenkins/plugins/k8sengine/KubernetesEnginePublisher.java

```diff
@@ -73,6 +76,7 @@
   private String credentialsId;
   private String zone;
   private String entryMethod;
+  private String projectIdEntry;
```

Comment:
Would prefer keeping the field naming consistent with existing fields. Suggest removing the "Entry" suffix.

Reply:
"projectIdEntry" is the analog to "entryMethod" for zone, but that name didn't have a lot of thought into it. Going to rename entryMethod to zoneEntry to make this clear.

Reply:
Understood, thanks for clarifying.